website/docs/r/nsxt_edgegateway_bgp_neighbor.html.markdown

```diff
@@ -0,0 +1,82 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_nsxt_edgegateway_bgp_neighbor"
+sidebar_current: "docs-vcd-resource-nsxt-edgegateway-bgp-neighbor"
+description: |-
+  Provides a resource to manage NSX-T Edge Gateway BGP Neighbors and their configuration.
+---
+
+# vcd\_nsxt\_edgegateway\_bgp\_neighbor
+
+Supported in provider *v3.7+* and VCD 10.2+ with NSX-T
+
+Provides a resource to manage NSX-T Edge Gateway BGP Neighbors and their configuration.
+
+## Example Usage (BGP Neighbor configuration with route filtering with referenced BGP IP Prefix List)
+
+```hcl
+resource "vcd_nsxt_edgegateway_bgp_neighbor" "neighbor-one" {
+  org = "datacloud"
+
+  edge_gateway_id = data.vcd_nsxt_edgegateway.testing.id
+
+  ip_address       = "1.1.1.1"
+  remote_as_number = "62513"
+
+  keep_alive_timer      = 78
+  hold_down_timer       = 400
+  graceful_restart_mode = "GRACEFUL_AND_HELPER"
+  allow_as_in           = false
+  bfd_enabled           = true
+  bfd_interval          = 800
+  bfd_dead_multiple     = 5
+
+  route_filtering              = "IPV4"
+  in_filter_ip_prefix_list_id  = data.vcd_nsxt_edgegateway_bgp_ip_prefix_list.in-1.id
+  out_filter_ip_prefix_list_id = data.vcd_nsxt_edgegateway_bgp_ip_prefix_list.out-1.id
```

Comment:
We have prefix here, but not in the test above.

Reply:
Yes. Tests don't have it because this PR needed other PR. Once it is all in `main` I will write additional tests that integrate these components together.